vars/buildImage.groovy

```diff
@@ -92,9 +92,9 @@ def makeImageStep(String pipeline_version, String arch, String debian_arch, Stri
                 }
 
                 stage("Upload images for ${arch}") {
-                    withCredentials([string(credentialsId: 'Staging KernelCI API Token', variable: 'API_TOKEN')]) {
```

Comment:
With the build and bisection jobs, I've set up a parameter so the token name can be stored in the job configuration:
https://github.com/kernelci/lava-ci/blob/master/jenkins/bisect.jpl#L203
```
        withCredentials([string(credentialsId: params.KCI_TOKEN_ID,
                                variable: 'SECRET')]) {
```
Same thing in the build job:
https://github.com/kernelci/kernelci-build/blob/master/jenkins/build.jpl#L98